code/modules/status_system/statusEffects.dm

```diff
@@ -2335,6 +2335,15 @@
 #undef LAUNDERED_COLDPROT_AMOUNT
 #undef LAUNDERED_STAIN_TEXT
 
+/datum/statusEffect/quickcharged
+	id = "quick_charged"
+	name = "Quick charged"
+	maxDuration = 7 MINUTES
+
```

Comment:
does this have an icon?

Reply:
Oh shoot forgot to add that uhh one second